app/views/applications/process/confirmation.html.slim

```diff
@@ -3,6 +3,8 @@ header
     .small-12.medium-12.large-12.columns
       h2 Processing complete
 
+= display_reference(@application)
```

Comment:
@novotnyjakub 
> This will have no formatting, will it?

Correct, discussed it with Kellie already and I mention it in the commit message: https://github.com/ministryofjustice/fr-staffapp/commit/028faf9432d0975864cefcd754d57555ba4d35f1

Once we know what the style should be applied, it'll be added.